matrix-sdk-crypto-nodejs: 0.3.0-beta.1 -> 0.4.0-beta.1, matrix-hookshot: 7.0.0 -> 7.1.0

christoph-heiss opened the pull request:
Release notes: https://github.com/matrix-org/matrix-hookshot/releases/tag/7.1.0
Git changelog: https://github.com/matrix-org/matrix-hookshot/compare/7.0.0...7.1.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

christoph-heiss committed:
matrix-hookshot: 7.0.0 -> 7.1.0

Release notes: https://github.com/matrix-org/matrix-hookshot/releases/tag/7.1.0
Git changelog: https://github.com/matrix-org/matrix-hookshot/compare/7.0.0...7.1.0

Signed-off-by: Christoph Heiss <christoph@c8h4.io>

chvp reviewed (changes requested):
The version of matrix-sdk-crypto-nodejs should also be bumped for this. Nixpkgs is on 0.3.0-beta1, but this now depends (indirectly) on 0.4.0-beta1: https://github.com/element-hq/matrix-bot-sdk/blob/main/package.json#L58

christoph-heiss committed:
matrix-sdk-crypto-nodejs: 0.3.0-beta.1-unstable-2025-02-11 -> 0.4.0-beta.1

Release notes: https://github.com/matrix-org/matrix-rust-sdk-crypto-nodejs/releases/tag/v0.4.0-beta.1
Git changelog: https://github.com/matrix-org/matrix-rust-sdk-crypto-nodejs/compare/v0.3.0-beta.1...v0.4.0-beta.1

Signed-off-by: Christoph Heiss <christoph@c8h4.io>

alyssais commented on pkgs/by-name/ma/matrix-sdk-crypto-nodejs/package.nix:
```diff
@@ -12,18 +12,18 @@
 stdenv.mkDerivation rec {
   pname = "matrix-sdk-crypto-nodejs";
-  version = "0.3.0-beta.1-unstable-2025-02-11";
+  version = "0.4.0-beta.1-unstable-2025-09-06";
```
Why is this still marked unstable?

alyssais replied:
That's my understanding.  It already says "beta" in the version.

alyssais reviewed (approved):
Looks fine to me, although I don't know any specifics about this package, so not sure why I was requested for review specifically.
